### ISSUE TITLE
Some problems in the demo

### ISSUE THREAD

**Issue by liiuzq-xiaobai** [WenjieDu/PyPOTS#57]
### Question description

![image](https://user-images.githubusercontent.com/70749713/232931602-27b95e48-1c51-4486-b166-6ad5f701c6ea.png)
I tried for the demo you have shown in the README.But It seems have some problems when run the model.How can I fix it? Thank you!

**Comment by WenjieDu** [comment]
Hi there 👋,

Thank you so much for your attention to PyPOTS! If you find PyPOTS helpful to your work, please star⭐️ this repository. Your star is your recognition, which can help more people notice PyPOTS and grow PyPOTS community. It matters and is definitely a kind of contribution to the community.

I have received your message and will respond ASAP. Thank you for your patience! 😃

Best,
Wenjie

**Comment by WenjieDu** [comment]
Hi, thank you very much for your feedback! I don't have local CUDA devices for testing, so this bug must be not detected now that testing only runs on CPUs. I'm going to find some available CUDA devices, then try to fix it. Stay tuned please. Will get back to you soon. Thank you!

**Comment by liiuzq-xiaobai** [comment]
ok! Thanks for your help!

**Comment by WenjieDu** [comment]
Fixed. Please reinstall PyPOTS with the code below:

```bash
git clone --depth 1 -b dev https://github.com/WenjieDu/PyPOTS
cd PyPOTS
pip install .
```

I'll release a v0.0.11 today to fix this bug in the current version v0.0.10. 
Thank you very much again for reporting this bug! If you think PyPOTS is useful, please star🌟 our repo, this means a lot to PyPOTS community!

**Comment by WenjieDu** [comment]
@liiuzq-xiaobai Could you please test on your side to confirm this issue has been solved? 😊 Many thanks!
